### ISSUE TITLE
Website needs auto-follow and re-focus buttons

### ISSUE THREAD

**Issue by Lewis8379** [bilde2910/Hauk#181]
Having just tried the demo, I'm quite disappointed that there's no ability to have the web interface follow the tracked movements. 

However there's also no button re-focus the view.

In other words, if you're tracking something in London, then you swipe in error and you're looking at Spain, you'd have to manually zoom out, find the icon, and zoom in.

I personally consider those quite basic things, so I'm quite surprised it isn't there?

On another note there should be options to disable the location buttons on the website for those of us who won't have the end user on the website/map needing to share their location back.

**Comment by Lewis8379** [comment]
Ah okay, I misunderstood the options by clicking the person icon, but I do find it odd that the center functionaliy is under a menu. Should be a button directly on the interface. The "show on map" could also be renamed to "Refocus location on map".

The "navigate to" option opens a geo code, which while my phone understands, my browser on my computer does not. Maybe an option to adjust that to a Google Maps link instead wouldn't go amiss.

As for showing my location, interesting, thanks, but that is a rather confusing 'feature'. If anything, that should be under a menu where it's title clearly describes what it does, because it certainly isn't obvious now :P

**Comment by licaon-kter** [comment]
The map is based on OpenStreetMaps... so maybe it should open that, instead of privacy invading Google, imho.

**Comment by rustyduffer** [comment]
Yeah I'm happy for fewer links between me and google whenever possible.
